documents/manifesto.md

```diff
@@ -0,0 +1,28 @@
+# The Lua Users Foundation Manifesto
+
+The Lua Users Foundation (the Foundation) is an association of individuals in a modular umbrella structure consisted of several workgroups, detailed in the Lua Users Foundation Constitution.
+The Foundation formal decision-making process is governed by the Lua Users Foundation Constitution, according to the Foundation Goals and Values, towards our Mission.
+
+## The Mission
+
+  * Support and promote the Lua programming language, its community and ecosystems.
+    
+## The Values
+
+  * The Lua programming language is developed by the Lua Team and defined by the Lua Programming Manual. Furthermore:
+  * The Lua programming language is small, fast, easy to learn and use. 
+  * The Lua programming language philosophy is less is more.
```

Comment:
I would write:

> The Lua programming language philosophy is: "Less is more".

Also I am not sure if this is not controversial...

Reply:
I like your phrasing. It probably is somewhat controversial, yes. But it is the original philosophy of Lua and I think we should strive not to deviate from that.

Reply:
I think the philosophy of lua is better described as "mechanism, not policy"

Reply:
> I think the philosophy of lua is better described as "mechanism, not policy"

Finally found a good citation from the lua team. https://www.lua.org/wshop13/Ierusalimschy.pdf see 2nd last page.

Reply:
Many thanks!